src/generate/members.rs

```diff
@@ -131,15 +147,94 @@ impl Writable for CppMethod {
     fn write(&self, writer: &mut super::writer::CppWriter) -> color_eyre::Result<()> {
         writeln!(
             writer,
-            "// Method: name: {}, Return Type Name: {} Parameters: {:?}",
-            self.name, self.return_type, self.parameters
+            "// Method: name: {}, Return Type Name: {} Parameters: {:?} Addr {:x} Size {:x}",
+            self.name,
+            self.return_type,
+            self.parameters,
+            self.method_data.addrs,
+            self.method_data.estimated_size
         )?;
 
         Ok(())
     }
 }
 impl Writable for CppProperty {
     fn write(&self, writer: &mut super::writer::CppWriter) -> color_eyre::Result<()> {
+        writeln!(
+            writer,
+            "// Property: name: {}, Type Name: {}, setter {} getter {} abstract {}",
+            self.name,
+            self.ty,
+            self.setter.is_some(),
+            self.getter.is_some(),
+            self.abstr
+        )?;
+
+        // TODO:
+        if self.abstr {
+            return Ok(());
+        }
+
+        if self.instance {
+            writeln!(
+                writer,
+                "::bs_hook::InstanceProperty<{},\"{}\",{},{}> {};",
+                self.name,
+                self.ty,
+                self.getter.is_some(),
+                self.setter.is_some(),
+                self.name
+            )?;
+        } else {
+            writeln!(
+                writer,
+                "static inline ::bs_hook::StaticProperty<{},\"{}\",{},{}, {}> {};",
+                self.ty,
+                self.name,
+                self.getter.is_some(),
+                self.setter.is_some(),
+                self.classof_call,
+                self.name
+            )?;
+        }
+
+        if self.generate_legacy {
```

Comment:
these are already present in the methods list, if you truly want to not create them, you will need to not the setter/getter methods for these props.